packages/arb-token-bridge-ui/tests/e2e/specs/withdrawCctp.cy.ts

```diff
@@ -99,13 +98,11 @@ describe('Withdraw USDC through CCTP', () => {
     cy.wait(10_000)
     cy.confirmMetamaskTransaction(undefined)
     const txData = {
+      duration: 'Less than a minute',
       amount: USDCAmountToSend,
       symbol: 'USDC'
     }
-    cy.findTransactionInTransactionHistory({
-      duration: 'a minute',
-      ...txData
-    })
+    cy.findTransactionInTransactionHistory(txData)
```

Comment:
`openTransactionDetails` only takes `amount` and `symbol`?
why is this changed on this PR anyway?